.gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+stages:
+  - build
+
+trigger_testbed:
+  stage: build
+  image:
+    name: registry.gitlab.sensirion.lokal/sensirion/docker/docker-ubuntu:18.04-1.0.0
+  tags:
+    - docker
+    - linux
+  script:
+  - >
+    curl
+    --request POST
+    --form token=$CI_JOB_TOKEN
+    --form ref=sts-test
```

Comment:
This needs to be changed to master at a later stage.

Reply:
Yes, but it is needed for testing the pipeline setup. After merge this should be set to 'master'.